anvil/src/udev.rs

```diff
@@ -304,95 +296,97 @@ fn scan_connectors(
         let encoder_infos = connector_info
             .encoders()
             .iter()
-            .filter_map(|e| *e)
-            .flat_map(|encoder_handle| device.get_encoder(encoder_handle))
+            .flatten()
+            .flat_map(|encoder_handle| device.get_encoder(*encoder_handle))
             .collect::<Vec<EncoderInfo>>();
-        'outer: for encoder_info in encoder_infos {
```

Comment:
This function looks like a massive delta, but I haven't changed much. It's mostly just less indentation which makes it possible to get rid of ugly things like loop annotations.